src/xdocs/config_imports.xml

```diff
@@ -1423,6 +1423,17 @@ import java.util.stream.IntStream;
             <td>false</td>
             <td>3.2</td>
           </tr>
+          <tr>
+          <td>separatedStaticGroups</td>
+              <td>
+                whether <b>static imports</b> grouped by <b>top</b> or <b>bottom</b> option should
+                be separated by, at least, one blank line or comment and aren't separated
+                internally.
```

Comment:
brrr .... complete confusion.

in issue we have 
> So, we can add a option named separatedStaticImports which means whether static imports grouped by top or bottom option are separated by local groups.

so this option suppose to care only about separation between static groups and do not rule how statics are separated from simple imports. Please read issue description one more time. Am I missing smth ?
Separation between static and simple import is ruled by `separated` option:
```
$ cat Test.java 
import static org.mockito.Matchers.any;
import com.util.Arrays2;
import java.util.Arrays;

public class Test {
}
rivanov@p5510:/var/tmp$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
      <module name="ImportOrder">
        <property name="option" value="top"/>
        <property name="separated" value="true"/>
      </module>
    </module>
</module>

$ java -jar checkstyle-8.10.1-all.jar -c config.xml Test.java 
Starting audit...
[ERROR] /var/tmp/Test.java:2: 'com.util.Arrays2' should be separated from previous imports. [ImportOrder]
Audit done.
Checkstyle ends with 1 errors.

$ cat config.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>

    <module name="TreeWalker">
      <module name="ImportOrder">
        <property name="option" value="top"/>
        <property name="separated" value="false"/>
      </module>
    </module>
</module>

$ java -jar checkstyle-8.10.1-all.jar -c config.xml Test.java 
Starting audit...
Audit done.

```